go.mod

```diff
@@ -1,3 +1,20 @@
 module github.com/rollkit/avail-da
 
-go 1.21.0
+go 1.21.1
```

Comment:
We should add a README with this PR explaining how to use the interface, including the required go version.